mrmap/main/tables/template_code.py

```diff
@@ -106,6 +106,13 @@
     {% if "delete_"|add:model_name in perms and record.get_delete_url %}
     <a href="{{record.get_delete_url}}" class="btn btn-sm btn-danger" data-toggle="tooltip" data-placement="left" title="{% trans 'Delete' %}">{{ ICONS.DELETE|safe }}</a>
     {% endif %}
+    {% if record|get_validate_url %}
+    <form class="mr-1" action="{{ record|get_validate_url }}" method="post">
```

Comment:
principle ok. But if we use post here, we get always the rendered view with a invalid form and error messages, cause the conformity_check_configuration field is required. Better would be a simple link with initial values:
```python
<a href="{{record|get_validate_url}}?metadata={{record.pk}}">{{ ICONS.VALIDATION }}</a>
```